### ISSUE TITLE
(FP8 Bug) handle long quotes gracefully

### ISSUE THREAD

**Issue by sjasthi** [sjasthi/AutoCap#33]
If you bump into a quote that is large (which can not be fit into the grid), handle it gracefully.

Ignore it and create an empty slide (so as to ensure that we are NOT messing up with the slide / quote numbers)

**Comment by sjasthi** [comment]
What is defined as "long quote" for 16x12 puzzle?

**Comment by Xeng651** [comment]
> What is defined as "long quote" for 16x12 puzzle?

Since we're using the characters as-is after processing through the getlogicalChars API, the length for the Telugu language also got shorten so it's able to generate puzzles for more quotes. (We implemented our program to use the length given through the API). 

Tested program. I believe a "long quote" for a 16x12 puzzle will be quotes of length 100 characters and above. The program will take forever to run because it's either due to the quote can not fit in the grid or just take a very slow/long time.  

Updated:
Actually, it's any **quote of length 70 characters and above.**
